### PR TITLE
Working Group Notice Board Structure

### PR DESCRIPTION
**Pull request by Sukriti-sood**
Signed-off-by: Sukriti-sood <soodsukriti1111@gmail.com>

# Pull Request Template

## Description
Added Working Group Notice Board on Community Section of community-website

## Type of change

- [x] New feature (non-breaking change which adds functionality).

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

ezgif com-gif-maker (2)

@chicken-biryani @HarshCasper

### CODE REVIEW

**Comment by chicken-biryani** [comment]
@Sukriti-sood thank you! A good movie is to add screenshots/videos in the PR.

@YashKandalkar @SanjaySinghRajpoot @sohamsshah we decided on the design-- Sukriti has implemented-- during the UI working group meeting. Can you please have a look as well?

**Review by SanjaySinghRajpoot — APPROVED** [review]

**Comment by SanjaySinghRajpoot** [comment]
Looks good to me.

**Review by YashKandalkar — APPROVED** [review]
LTGM!

**Review by chicken-biryani — APPROVED** [review]

**Comment by chicken-biryani** [comment]
@Sukriti-sood @SanjaySinghRajpoot @YashKandalkar a question tho.

How do you think we should manage the meeting notes? It can go date wise and linked. But once linked... Where will the link lead to?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few observations:

- Why do we have so many docs for working groups that have not even started right now? 
- I'm not sure why do we have empty docs right here. The content should be imported from `About_moja_global` before we merge them.
- Why do we have `Agenda` and `Meeting notes` on the community website? Aren't they already being published on `About_moja_global`? Why do we need to duplicate the content?
- I'm not sure about the position of working groups in the sidebar. 

The PR is not in a state to merge until the above comments are addressed.

**Comment by chicken-biryani** [comment]
Hi, @HarshCasper

The notes are in About moja Global repo temporarily.

The idea was to be ready with the content and layout together so that we develop things faster. Once the layout's ready... The contents will be shifted. And deleted from About moja global.

I do agree. Maybe we should keep the working groups limited to the ones that are currently active.

We agreed on the sidebar in yesterday's meet. We would be happy to hear your advice on how can we improve?

**Comment by HarshCasper** [comment]
@chicken-biryani 

It doesn't make sense to me. We are refactoring `About_moja_global` to host content that is dynamically changing, like meeting notes, progress and more. Using it for Community Website defeats that purpose.

At best, we can just link meeting notes, agenda and more to `About_moja_global` and rather keep the Community Website as an official word about the working groups. Even the primary sources, you are taking inspiration from, does not host meeting notes on an official website.

Coming to Sidebar, the first focus is on getting folks started up and letting them know how to contribute. The working groups should follow later.

**Comment by chicken-biryani** [comment]
@HarshCasper you're right. My bad... It was my idea to shift things. Thanks for reflecting!

@Sukriti-sood can you do the following:

(@HarshCasper do correct me if I am wrong anywhere)

- We're just going to add the About section to the community website. So We can let go the meeting notes and Agenda.

- Shift the 'working-groups' below the contribution section.

- Remove the working groups that aren't active. i.e keep UI and Documentation working group.

- Take the content of About-UI from the working groups folder and  About- Documentation from Harsh.

@HarshCasper if there are more required changes or changes please do add them.

**Comment by Sukriti-sood** [comment]
@chicken-biryani  I will update the pr with mentioned changes.

**Comment by Sukriti-sood** [comment]
@chicken-biryani @HarshCasper Should the any-Working group section include links to Agendas and Meeting Notes?.?

**Comment by chicken-biryani** [comment]
Hi @Sukriti-sood can you please put a video with the updates!

Thanks.

**Comment by Sukriti-sood** [comment]
ezgif com-gif-maker (3)

As discuused in the last ui-wg meeting, Arpita will add UI-wg documentaton, I have not added.

**Comment by chicken-biryani** [comment]
@Sukriti-sood I think Arpita can take up that task for another upcoming working group?

Can you please add info for the UI working group as well. 😅

**Comment by Sukriti-sood** [comment]
ezgif com-gif-maker (4)

**Comment by chicken-biryani** [comment]
Thank you so much, @Sukriti-sood!

@sohamsshah and @HarshCasper do share your opinions.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: website/community/working-groups/doc-wg.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+id: doc-wg
+title: Documentation Working Group
+---
+
+![Active Working Group](https://img.shields.io/badge/STATUS-ACTIVE-green?style=for-the-badge)
```
[the remark this answers is not in the document]
Comment (Sukriti-sood) [reply website/community/working-groups/doc-wg.md:6]: Yes, will update this.

**Comment by aornugent** [comment]
Nice work @Sukriti-sood!

Can we please add the working groups to the Getting Started section of 'Know the community' and the footer of the main page? Maybe even the top level of the side bar, under Projects and above Blogs.

**Comment by Sukriti-sood** [comment]
@aornugent I have added link in footer and `know the community` Section. As of now, it is directed to `documentation working group section` but later we can link it to `What are working Groups` Section (Still not added). But Currently I don't think its a good idea to add `working-groups` Section  to sidebar.

**Comment by aornugent** [comment]
@Sukriti-sood - are you ready to merge? 🤞

**Comment by Sukriti-sood** [comment]
> @Sukriti-sood - are you ready to merge? crossed_fingers

Yes, If no changes required 😅

**Pull request merged**